tests/test_pandas_integration.py

```diff
@@ -22,10 +22,31 @@
 
 @pytest.fixture
 def test_array_chunked():
-    chunks = []
-    for _ in range(10):
-        chunks.append(pa.array(TEST_LIST))
-    return pa.chunked_array(chunks)
+    return pa.chunked_array([pa.array(TEST_LIST) for _ in range(10)])
+
+
+@pytest.fixture(
+    params=["all", "all_float", "some_in_all_chunks", "only_in_some_chunk", "none"]
+)
+def test_array_chunked_nulls(request):
```

Comment:
Please don't prefix a fixture with `test_` as this isn't a test case.

Reply:
Done.